agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -217,23 +217,6 @@ bool slave_thread::socket_disconnected(Socket *sd)
         return true;
     }
 
-    auto ap_manager_err_code_to_bpl_err_code = [](int code) -> int {
-        if (code == ap_manager_thread::eThreadErrors::APMANAGER_THREAD_ERROR_HOSTAP_DISABLED) {
-            return BPL_ERR_AP_MANAGER_HOSTAP_DISABLED;
```

Comment:
These values are no longer used so should be removed from bpl_err.h as well.

Reply:
I know, I decided to keep them because I thought we will want to notify about it.
Anyhow, if @tomer will decide to remove the bpl_notification they will be removed anyway.